source/user/sgx.rst

```diff
@@ -0,0 +1,52 @@
+.. _user_sgx:
+
+Seminumerical Exchange (SGX)
+****************************
+
+*Modules*: :mod:`sgx`
+
+The SGX module implements seminumerical computation of the exchange matrix.
+
+Introduction
+============
+
+Direct computation of the Hartree-Fock Exchange matrix in the atomic orbital basis scales poorly with system size. To achieve better scaling, one coordinate of the two-electron integrals is computed analytically, while the other is evaluated on a real-space grid, as proposed by Friesner :cite:`Friesner1985`. The PySCF implementation most closely resembles the chain-of-spheres (COSX) algorithm of Neese et al. :cite:`Neese2009`, but with more conservative grids and without P-junction screening. Overlap fitting is used to reduce aliasing errors :cite:`Izsak2011`. SGX scales as :math:`O(N^2)` with system size, as opposed to the :math:`O(N^4)` scaling of analytical exchange :cite:`Neese2009`.
+
+Usage and Example
+=================
+
+Any :attr:`scf.hf.SCF` object :attr:`mf` can be converted to an equivalent object that computes the Coulomb and Exchange matrices with SGX instead of analytical integration by calling :code:`sgx.sgx_fit(mf)`.
+
+* :source:`examples/sgx/00-simple_sgx.py`
+
+.. literalinclude:: ../../examples/sgx/00-simple_sgx.py
+
+.. code::
+
+  converged SCF energy = -76.0267374704185
+  converged SCF energy = -76.0267978618974
+
+In this case, the error of SGX compared to analytical exchange is about 0.06 mH. The line
+
+.. code::
+
+    mf.with_df.dfj = True
+
+specifies to compute the Coulomb matrix using Resolution of the Identity (RI) while using SGX for the Exchange matrix.
```

Comment:
I don't have an opinion on this but isn't this another phrase for density fitting (I might be wrong on this) which seems to be the terminology used in pyscf? Please correct me if I am wrong or ignore if you prefer this phrasing.

Reply:
I agree that DF is clearer here though; I will change to DF.

Reply:
Resolved by 4c4a5431f26b876fd5b50f9bfededef8e840cb5f